### PR TITLE
Chore/update ds to newest

### PR DESCRIPTION
**Pull request by sonwit**
Update DS to the latest version 
## Description
- updated Modal to new DS
- made new reusable Pagination component based on the updated DS pagination
- Updated the chips to use checkbox instead of toggle
- removed Chip.Group
- Removed unsupported spacing prop from header and paragraph and added `variant="long"` or manual spacing where needed
- Replaced `<Ingress>` with `<Paragraph variant="long" />` according too DS recommendations  
- Added more stories for testing 

## Related Issue(s)
NA

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Changes
- Chip component looks different  (new left / old right)
Skjermbilde 2024-10-22 kl  13 00 08
Skjermbilde 2024-10-22 kl  12 59 15
- Modal.Block has a border now
Skjermbilde 2024-10-22 kl  13 05 54
Skjermbilde 2024-10-22 kl  13 06 01

### CODE REVIEW

**Review by allinox — APPROVED** [review]
Great work! 😄

I have only a few suggestions, and they are all related to styling and not anything critical. Approving this so you can just merge when you've taken a look at them 😊

File: src/features/amUI/userRightsPage/DelegationModal/DelegationModalContent.tsx [hunk 1]
```diff
@@ -21,36 +21,43 @@ export const DelegationModalContent = ({ toParty }: DelegationModalProps) => {
...
+      <Modal
         className={classes.modalDialog}
-        onInteractOutside={closeModal}
+        backdropClose
+        closeButton={t('common.close')}
         onClose={onClose}
+        ref={modalRef}
       >
-        <Modal.Header>
+        <Modal.Block>
```
[the remark this answers is not in the document]
Comment (sonwit) [reply src/features/amUI/userRightsPage/DelegationModal/DelegationModalContent.tsx:60]: 👍

File: src/components/ActionBar/ActionBarContent.module.css
[the remark this answers is not in the document]
Comment (sonwit) [reply src/components/ActionBar/ActionBarContent.module.css:1]: 👍🏼

File: src/features/singleRight/delegate/ChooseServicePage/ChooseServicePage.tsx [hunk 2]
```diff
@@ -85,11 +85,11 @@ export const ChooseServicePage = () => {
                 />
               ) : (
                 <>
-                  <Ingress spacing>
+                  <Paragraph variant='long'>
```
[the remark this answers is not in the document]
Comment (sonwit) [reply src/features/singleRight/delegate/ChooseServicePage/ChooseServicePage.tsx]: 👍

**Pull request merged**